Makefile

```diff
@@ -108,7 +134,8 @@ ifneq (,$(findstring clever-cloud,$(PGHOST)))
 else
 	dropdb ${PGDATABASE} || true
 	createdb ${PGDATABASE}
-	time pg_restore --format=directory --clean --jobs=4 --verbose --no-owner --no-acl -d ${PGDATABASE} $(RESTORE_DIR) || true
+	time pg_restore --format=directory --clean --jobs=4 --verbose --no-owner --no-acl -d ${PGDATABASE} \
+		$(RESTORE_DIR) || true
```

Comment:
_nit_, normalement pas besoin de modifier cette ligne :
```suggestion
	time pg_restore --format=directory --clean --jobs=4 --verbose --no-owner --no-acl -d ${PGDATABASE} $(RESTORE_DIR) || true
```

Reply:
ok mais pas besoin de m'insulter... 
![image](https://github.com/user-attachments/assets/0f96c2a1-ddcc-4cd0-b6b2-0b8f048784e1)

Reply:
Ah ! :see_no_evil: 
https://stackoverflow.com/questions/27810522/what-does-nit-mean-in-hacker-speak